spec.md

```diff
@@ -2147,6 +2156,50 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 |-----------|-----------|-------------|-------------------|
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListSnapshots` operation again with an empty `starting_token`. |
 
+#### `GetSnapshot`
+
+**ALPHA FEATURE**
+
+This optional RPC MAY be called by the CO to fetch current information about a snapshot.
+
+A Controller Plugin MUST implement this `GetSnapshot` RPC call if it has `GET_SNAPSHOT` capability.
+
+SP supporting Group Controller Service and the CREATE_DELETE_GET_VOLUME_GROUP_SNAPSHOT capability SHOULD implement the `GetSnapshot` RPC in order to retrieve information for individual snapshots that are part of a group snapshot.
+
+`GetSnapshotResponse` should contain current information of a snapshot if it exists.
+If the snapshot does not exist any more, `GetSnapshot` should return gRPC error code `NOT_FOUND`.
```

Comment:
Should we also say that snapshots which were recently created and not yet ready_to_use must be retrievable with this RPC? I think it's kind of obvious, but it would be good to change the current mechanism of polling for ready_to_use to rely on GetSnapshot instead of CreateSnapshot for SPs that support it in the future. Having some language here would make us feel safer about making that change.

Reply:
Done